ifbcat_api/model/event.py

```diff
@@ -243,8 +244,11 @@ class EventAccessibilityType(models.TextChoices):
     )
 
     def clean(self):
-        if self.accessibility == self.EventAccessibilityType.PRIVATE and len(self.accessibilityNote or '') == 0:
-            raise ValidationError(dict(accessibilityNote="Details have to be provided when accessibility is private"))
+        errors = {}
+        if self.open_to == self.EventOpenToType.OTHERS and len(self.access_conditions or '') == 0:
```

Comment:
I think the conditions have to be specified when event is not EVERYONE, not just when type is OTHERS